OracleAccessManagement/imagetool/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,401 @@
+```
+
+### ii) Add installers to Imagetool cache
+
+```bash
+$ ./imagetool.sh cache addInstaller --type fmw --version 12.2.1.4.0 --path <work directory>/stage/fmw_12.2.1.4.0_infrastructure.jar
+$ ./imagetool.sh cache addInstaller --type OAM --version 12.2.1.4.0 --path <work directory>/stage/fmw_12.2.1.4.0_idm.jar
+```
+
+### iii) Add Patches to Imagetool cache
+
+```bash
+$ ./imagetool.sh cache addEntry --key 28186730_13.9.4.2.4 --value <work directory>/stage/p28186730_139424_Generic.zip
+$ ./imagetool.sh cache addEntry --key 31556630_12.2.1.4.0 --value <work directory>/stage/p31556630_122140_Generic.zip
+$ ./imagetool.sh cache addEntry --key 31537019_12.2.1.4.0 --value <work directory>/stage/p31537019_122140_Generic.zip
+$ ./imagetool.sh cache addEntry --key 31544353_12.2.1.4.0 --value <work directory>/stage/p31544353_122140_Linux-x86-64.zip
+$ ./imagetool.sh cache addEntry --key 31470730_12.2.1.4.0 --value <work directory>/stage/p31470730_122140_Generic.zip
+$ ./imagetool.sh cache addEntry --key 31488215_12.2.1.4.0 --value <work directory>/stage/p31488215_122140_Generic.zip
+```
+
+### iv) Add patches to the buildArgs file:
+
+Edit the `buildArgs` file and add the patches:
+
+```
+--patches 31556630_12.2.1.4.0,31488215_12.2.1.4.0,31470730_12.2.1.4.0,31537019_12.2.1.4.0,31544353_12.2.1.4.0
+--opatchBugNumber=28186730_13.9.4.2.4
+```
+
+A sample `buildAgs` file is now as follows:
+
+```
+create
+--jdkVersion=8u261
+--type oam
+--version=12.2.1.4.0
+--tag=oam-with-patch
+--pull
+--installerResponseFile /scratch/docker-images/OracleFMWInfrastructure/dockerfiles/12.2.1.4.0/install.file,/scratch/docker-images/OracleAccessManagement/dockerfiles/12.2.1.4.0/install/iam.response
+--additionalBuildCommands /scratch/docker-images/OracleAccessManagement/imagetool/12.2.1.4.0/addtionalBuildCmds.txt
+--additionalBuildFiles /scratch/docker-images/OracleAccessManagement/dockerfiles/12.2.1.4.0/container-scripts
+--patches 31556630_12.2.1.4.0,31488215_12.2.1.4.0,31470730_12.2.1.4.0,31537019_12.2.1.4.0,31544353_12.2.1.4.0
+--opatchBugNumber=28186730_13.9.4.2.4
+```
+
+### v) Create the OAM image
+
+Execute the `imagetool create` command to create the OAM image.
+
+For example:
+
+```bash
+$ cd <work directory>/imagetool/bin
+$ ./imagetool.sh @<work directory>/docker-images/OracleAccessManagement/imagetool/12.2.1.4.0/buildArgs
+```
+
+###  vi) View the docker image
+
+Run the `docker images` command to ensure the new OAM image is loaded into the repository:
+
+```
+$ docker images
+REPOSITORY                                                    TAG                 IMAGE ID            CREATED             SIZE
+oam-with-patch                                                12.2.1.4.0          d4cccfcd67c4        3 minutes ago      3.38GB
+oraclelinux                                                   7-slim              153f8d73287e        2 weeks ago         131MB
+```
+
+# 7. Sample Dockerfile generated with imagetool
```

Comment:
Why is this in the README.md? You provide the command to produce, so you don't need to replicate its content again.